src/quantum/azext_quantum/operations/templates/create-workspace-and-assign-role.json

```diff
@@ -95,14 +95,18 @@
                     "contentVersion": "1.0.0.0",
                     "resources": [
                         {
-                            "apiVersion": "2019-06-01",
```

Comment:
Although I wrote it in the ADO work item, this is actually not necessary, and we can leave it with the original value.